src/pages/test-report/test-report.scss

```diff
@@ -20,4 +20,10 @@ page-test-report {
     margin: 4px 8px;
   }
 
+  #test-report-header {
+    height: 68px;
+    padding: 12px 17px 12px 17px;
```

Comment:
this could be shorthanded to `padding: 12px 17px`

Reply:
good point